packages/react-query/src/BlockAuthors.tsx

```diff
@@ -39,12 +39,12 @@ function BlockAuthors ({ children }: Props): React.ReactElement<Props> {
       let lastBlockNumber = '';
 
       // subscribe to all validators
-      api.query.session && api.query.session.validators((validatorIds): void => {
-        setValidators(validatorIds.map((validatorId): string => validatorId.toString()));
+      api.query.session && api.query.session.validators((validatorIds: any): void => {
```

Comment:
Maybe these types
`AccountId[]`
`Header`

Reply:
I tried with those, I got error.. even polkadot/apps do not provide any types for these..